### PR TITLE
Popup window

### PR DESCRIPTION
**Pull request by sunga12**
Add javascript for featured project and popup window

### CODE REVIEW

**Review by omar-muhamad — COMMENTED** [review]
# Invalid ❌

Hi Team,

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

Your Code Review Request is marked as invalid for these reasons:

- [ ] Eslint linters GitHub action is not configured, Please use this [linters.yml](https://github.com/microverseinc/linters-config/blob/master/html-css-js/.github/workflows/linters.yml) file instead of the one you used to make it work because for this project you suppose to add linters actions for [HTML, CSS, and javascript.](https://github.com/microverseinc/linters-config/tree/master/html-css-js)

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **Does not count** into the code reviews limit._

File: script.js [hunk 1]
```diff
@@ -18,4 +18,336 @@ lists.forEach((list) => {
...
+    popupTitle.innerHTML = project.name;
+    projectPopup.appendChild(popupTitle);
+
+    const popupClose = document.createElement('img');
+    popupClose.className = 'popup-close';
+    popupClose.src = './images/close.png';
+    popupTitle.appendChild(popupClose);
+
+    const popupLangs = document.createElement('ul');
+    popupLangs.className = 'popup-buttons';
```
Comment (GzimAsani) [inline script.js:299]: - [ ] Kindly adjust the padding of buttons in order to match the figma design

Yours:
image

Design:
image

File: script.js [hunk 2]
```diff
@@ -18,4 +18,336 @@ lists.forEach((list) => {
...
+  popupTitle.innerHTML = featuredProject.name;
+  projectPopup.appendChild(popupTitle);
+
+  const popupClose = document.createElement('img');
+  popupClose.className = 'popup-close';
+  popupClose.src = './images/close.png';
+  popupTitle.appendChild(popupClose);
+
+  const popupLangs = document.createElement('ul');
+  popupLangs.className = 'popup-buttons';
```
Comment (GzimAsani) [inline script.js:169]: - [ ] Please implement styling for buttons, hover and pressed 
image

File: script.js (hunk 1 above)
Comment (GzimAsani) [reply script.js:299]: - [ ] Add paddings on the side and adjust the X button, check the images for more info

Yours:
image

Design:
image

**Review by GzimAsani — CHANGES_REQUESTED** [review]
### Hello @sunga12                     :wave:   
 Good job so far,  done There are some issues that you still need to work on to go to the next project but you are almost there!
 
 ## Highlights
 * Good job with adding javascript, it looks great :+1: 

 ## Required Changes 
- [ ] Check the comments below :arrow_double_down: 


 ## Optional suggestions
 _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding! clapclapclap
 
 Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.
 
 **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
 
 _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by sunga12** [comment]
Changes Required:

- Adjust the padding of the buttons in order to match the Figma design
- Add paddings on the side of the Image in the mobile popup window and adjust the X button according to Figma design
- Implement styling for buttons: hover, and pressed

Changes Made:

- Adjusted the padding of the buttons in order to match the Figma design
- Added paddings on the side of the Image in the mobile popup window and adjust the X button according to Figma design
- Implemented styling for buttons: hover and pressed

File: script.js (hunk 1 above)
Comment (mckent05) [reply script.js:299]: - Good job done so far, but as a result of the adjustments made following the last review, you seem to have tampered with your live buttons, please fix this 🙏. Remember it is important that you try to replicate the Figma design as closely as possible to give confidence to recruiters in your ability to produce exact business requirements 🙏

Your project:
ergergaerg

Figma design:
q3wtgq34t
- Also, as a result of the adjustments you made following the last review, you seem to have introduced a bug. Please fix this 🙏 
 Your project:
q34tq34t

Design template:
efQEFqe

**Review by mckent05 — CHANGES_REQUESTED** [review]
Hi @sunga12, 

# STATUS: CHANGES REQUIRED ♻️ ♻️ 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.
almost there


 ## Highlights
- All linter checks are passing ✔️ ✔️ 
- Your PR is professional 👍 👍 


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your code.


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by sunga12** [comment]
Changes made:

- Fixed bugs related to live buttons in popup window.

**Review by m05e5 — APPROVED** [review]
**Hi @sunga12 

You did great. Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
CleverScornfulBuffalo-size_restricted

## Highlights

✔️ Well done implementing the requested changes 

## Optional suggestions



_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

_**m05e5**_

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._**

**Pull request merged**